### ISSUE TITLE
OnBeamableReady not useable as event in Blueprint EventGraph after commit (0f1213ffb2702f1fe41f6291ce5a5bcf3e7feee7)

### ISSUE THREAD

**Issue by Tomiha** [beamable/UnrealSDK#16]
I noticed that the signature of the `BeamableRuntimeSubsystem::OnBeamableReady` changed from `void ` to `FBeamOperationHandle `return value. 
This causes the issue that the event can't be added to the EventGraph anymore as shown in the ReadMe file about setting up the Beamable Plugin. (Example with the GetMe flow)

Not sure if this is intentional?

**Comment by PedroRauizBeamable** [comment]
Yes --- we changed the way `OnBeamableReady` was supposed to work a couple of times during development. The recommended way to set things up there now is:

- On your game's boot-up process (wherever you put your game's initialization code), call the use `UBeamRuntime::RegisterOnReady` to register a function to run when login finishes. This will either:
  - Run the given callback immediately, if the first owner user has already signed in by the time you make the call.
  - Wait until the first owner user has fully signed in and then run the given callback.
- From that function, get your various `BeamRuntimeSubsystem` and call a function manually on them.

This approach leads to more robustness as we found the "several subsystems hooking up to a callback that can't provide ordering semantics"-approach was too error prone for initialization code and was very likely to introduce subtle bugs that were hard to find. So... we moved to an approach where we recommend game-makers to have a single entry-point and explicitly manage their "OnReady" flow which is typically a lot of cross-cutting concerns code with lots of tiny subtle dependencies between systems.
